DEVDOCS-5190: [update] Wallet Buttons, support themes <6.7 (#2017)
bigcommerce/dev-docs: docs/stencil-docs/page-builder/wallet-buttons.mdx

![wallet-buttons-04](https://storage.googleapis.com/bigcommerce-production-dev-center/images/wallet-buttons-04.png "theme-markup")

### For Cornerstone themes below version 6.7 

You'll need to perform another step to make wallet buttons work as expected for themes below version 6.7. After you enter the code snippet above, insert the following code into your theme markup.

> ```html

Reviewer: ```suggestion
```html showLineNumbers copy
```

![wallet-buttons-04](https://storage.googleapis.com/bigcommerce-production-dev-center/images/wallet-buttons-04.png "theme-markup")

### For Cornerstone themes below version 6.7 

> You'll need to perform another step to make wallet buttons work as expected for themes below version 6.7. After you enter the code snippet above, insert the following code into your theme markup to look as follows:

Reviewer 2: ```suggestion
In themes below version 6.7, you need to perform another step to make wallet buttons work as expected . After you enter the preceding code snippet, insert the following code into your theme markup:
```
